### PR TITLE
Fix some PHP code for the gleam-for-php-users guide

### PR DESCRIPTION
**Pull request by fitv**

### CODE REVIEW

**Review by lpil — APPROVED** [review]
Thank you

File: cheatsheets/gleam-for-php-users.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ pub fn distance(from x: Int, to y: Int) : Int {
   abs(x) - abs(y) |> abs()
 }
 // In some other function
-distance(from: 1, to: -2) // 3
+distance(from: 1, to: -2) // 1
```
Comment (inoas) [inline cheatsheets/gleam-for-php-users.md:294]: but why should the distance from 1 to -2 be 1?

Comment (inoas) [reply cheatsheets/gleam-for-php-users.md:294]: @fitv @lpil

Comment (inoas) [reply cheatsheets/gleam-for-php-users.md:294]: nvm... just a bad name

**Pull request merged**